packages/calypso-e2e/src/jest-playwright-config/global-setup.ts

```diff
@@ -7,6 +7,12 @@ import pwConfig from './playwright-config';
 export default async (): Promise< void > => {
 	const { AUTHENTICATE_ACCOUNTS } = envVariables;
 
+	// If PWDEBUG mode is enabled (stepping through each step)
+	// don't execute the cookie refresh.
+	if ( process.env.PWDEBUG ) {
+		return;
+	}
```

Comment:
I'm open to discussion on this one.

My line of thinking was that:
- If I'm using PWDEBUG mode to step through a script
- then I probably do not need or want to deal with the three cookie saving prompts